### ISSUE TITLE
Ask user to stop runer when closing runner tab

### ISSUE THREAD

**Issue by m3dbedb** [a85/POSTMan-Chrome-Extension#10377]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

User may close runner tab while collection execution. You may still see execution output in log. If something went wrong and execution went in unstoppable loop, you cant termitate it with GUI.

### Steps To Reproduce

1.Select collection with unconditional transition to previous step (to imitate something went wrong in exucution and runner don't stop).
2.Run collection
3.Close runner tab.
4.You ca see execution log and can't stop it.

### Environment Information

```markdown
- Operating System:  windows 7 32
- Platform Type: i5
- Postman Version: 8.6.0
```

### Additional Context?

It woul be great if postman show warning message "you are going to close runner tab and if execution would loop forever you would not be able to terminate it unless you close postman at all".

**Comment by malvikach** [comment]
@m3dbedb thanks for writing to us and letting us know about this issue. Our team will be looking into this and fixing the same.

**Comment by malvikach** [comment]
Hi @m3dbedb  👋

Glad to update that the above has been fixed in the latest Postman version (10.6.*). If you accidentally close the tab now, you will be asked to confirm if you wish to stop the run or continue it.

![Screenshot 2022-12-08 at 11 08 02 PM](https://user-images.githubusercontent.com/89449769/206526294-ebcbab3a-8e9b-41f1-91fe-c63f06729fcf.png)

Hope this helps in taking an informed decision about the run execution. I am closing this issue, please feel free to let us know if you have any other feedback. Thanks again for suggesting this improvement. 

Cheers!
Malvika
